src/device-state/index.ts

```diff
@@ -790,9 +815,20 @@ export function triggerApplyTarget({
 			}
 			lastApplyStart = process.hrtime();
 			log.info('Applying target state');
-			return applyTarget({ force, initial });
+			if (!abortController) {
```

Comment:
Can we make it so abortController is never null? I don't like these Inconsistency errors if we can avoid them

Reply:
Is this still the case with https://github.com/balena-io-modules/docker-progress/pull/76?

Reply:
I mean making sure the variable in the supervisor is never null, i.e. assigning a value to it when the variable is created, so we don't have to keep checking it